gooddata-sdk/gooddata_sdk/catalog/workspace/service.py

```diff
@@ -78,6 +83,49 @@ def delete_workspace(self, workspace_id: str) -> None:
             )
         self._entities_api.delete_entity_workspaces(workspace_id)
 
+    def get_declarative_workspace(self, workspace_id: str) -> CatalogDeclarativeWorkspaceModel:
```

Comment:
This is just a note. I would expect here class named `CatalogDeclarativeWorkspace` instead of `CatalogDeclarativeWorkspaceModel`. But it is following naming convention of OAPI schema structures so lets keep it.